MN/Algoritmi/Valori-Proprii/metoda_puterii_inverse.m

```diff
@@ -0,0 +1,29 @@
+function [lambda, step] = metoda_puterii_inverse(A, tol, miu, max_iters)
```

Comment:
- InversePowerMethod in loc de metoda_puterii_iinverse
- ordinea parametrilor: A, tol, max_iter (nu max_iters), miu (ca sa fie primele 3 ca la PowerMethod)
- returnezi si vectorul asociat lui lambda, deci [lambda, y, step]